src/domUpdates.js

```diff
@@ -224,11 +224,19 @@ const updateSaveButtons = (ID, addButton, removeButton) => {
 }
 
 const populateRecipeHeader = currentRecipe => {
+  let filteredTags = filterTagsByTagName(pageData.allTags, currentRecipe.tags);
+  console.log(JSON.stringify(pageData.allTags))
+  let recipeTagsHTML = filteredTags.map((tag) => {
+    tag.isActive = true;
+    return createTagCardHTML(tag);
+  });
+
   document.querySelector('#recipeName').innerHTML = `
-  <h1>${currentRecipe.name}</h1>
-  <div class="individual-recipe-image">
-    <img src="${currentRecipe.image}"></img>
-  </div>
+    <div id="individualRecipeTags">${recipeTagsHTML.join('')}</div>
+    <h1>${currentRecipe.name}</h1>
+    <div class="individual-recipe-image">
```

Comment:
So if we're going for displaying tags without the tag icons (from the main page), this could be a different class which has a certain height, auto width and the orange background?

Reply:
yah we could def go with a different class if that's the case.  Are the main page tag colors changing at all tho?

Reply:
They're not.

Reply:
@mbenfowler Could we also align the tags to the left? In my screen, they're getting clipped by the recipe image.